src/instruments/src/SD/Pages/Fctl/Fctl.tsx

```diff
@@ -1,738 +1,390 @@
+    const hydraulicAvailableClass = hydraulics.G.available || hydraulics.Y.available ? 'Value' : 'Warning';
 
-const ElacSecShape = ({ id, x, y, number, on = true, fail = false } : ElacSecShapeProps) => {
-    const textPositionX = x + 61;
-    const textPositionY = y - 12;
     return (
-        <>
-            <path className={on && !fail ? 'MainShape' : 'MainShapeWarning'} d={`M${x},${y} l72,0 l0,-26 l-8,0`} />
-            <text
-                id={id}
-                className={on && !fail ? 'Value Standard' : 'Warning Standard'}
-                x={textPositionX}
-                y={textPositionY}
-                textAnchor="middle"
-                alignmentBaseline="central"
-            >
-                {number}
+        <SvgGroup x={x} y={y}>
+            <Note x={0} y={13}>PITCH TRIM</Note>
+            <text x={1} y={35} className={`${hydraulicAvailableClass} Standard`} textAnchor="end" alignmentBaseline="central">{pitchIntegral}</text>
+            <text x={5} y={35} className={`${hydraulicAvailableClass} Standard`} textAnchor="middle" alignmentBaseline="central">.</text>
+            <text x={14} y={37} className={`${hydraulicAvailableClass} Small`} textAnchor="middle" alignmentBaseline="central">{pitchFractional}</text>
+            <text x={28} y={35} className="ValueCyan Standard" textAnchor="middle" alignmentBaseline="central">°</text>
+            <text x={48} y={37} className={`${hydraulicAvailableClass} Small`} textAnchor="middle" alignmentBaseline="central">
+                {Math.sign(adjustedPitchTrim) === -1 ? 'DN' : 'UP'}
             </text>
-        </>
+
+            <HydraulicIndicator x={80} y={0} type="G" />
+            <HydraulicIndicator x={102} y={0} type="Y" />
+        </SvgGroup>
     );
 };
 
-type AileronAxisProps = {
-    leftorright: string,
-    x: number
-}
+const Rudder = ({ x, y }: ComponentPositionProps) => {
+    const [rudderDeflectionState] = useSimVar('RUDDER DEFLECTION PCT', 'percent over 100', 50);
+    const rudderAngle = -rudderDeflectionState * 25;
 
-const AileronAxis = ({ leftorright, x } : AileronAxisProps) => {
-    const d1 = `M${x},164 l${
-        leftorright === 'left' ? '-' : ''}8,0 l0,-20 l${
-        leftorright === 'right' ? '-' : ''}8,0 l0,120 l${leftorright === 'left' ? '-' : ''}8,0 l0,-10 l${leftorright === 'right' ? '-' : ''}8,0`;
-    const d2 = `M${x},200 l${leftorright === 'left' ? '-' : ''}7,0`;
-    const d3 = `M${x},205 l${leftorright === 'left' ? '-' : ''}7,0`;
-    const d4 = `M${x},210 l${leftorright === 'left' ? '-' : ''}8,0 l0,6 l${leftorright === 'right' ? '-' : ''}8,0`;
+    // Rudder limits
+    const [indicatedAirspeedState] = useSimVar('AIRSPEED INDICATED', 'knots', 500);
+    let maxAngleNorm = 1;
+    if (indicatedAirspeedState > 380) {
+        maxAngleNorm = 3.4 / 25;
+    } else if (indicatedAirspeedState > 160) {
+        maxAngleNorm = (69.2667 - 0.351818 * indicatedAirspeedState
+            + 0.00047 * indicatedAirspeedState ** 2) / 25;
+    }
```

Comment:
@aguther this piece of JS code calculates the displayed maximum deflection of the rudder on the F/CTL page. As far as I can tell it doesn't match with the maximum deflection as managed by the FBW implementation. Is this something you could expose in the form of an LVar so it can be used here?

Reply:
I need to recheck if and how I implemented this, there is another limitation in the flight model as far as I know. Will check tomorrow and let you know.